src/main/java/uk/gov/hmcts/sptribs/ciccase/model/SubjectCIC.java

```diff
@@ -8,11 +8,10 @@
 @Getter
 @AllArgsConstructor
 public enum SubjectCIC implements HasLabel {
-    @JsonProperty("Subject")
+    @JsonProperty("SubjectCIC")
     SUBJECT("Subject");
-
     private final String label;
     public boolean isSubject() {
-        return true;
+        return SUBJECT.name().equalsIgnoreCase(this.name());
     }
```

Comment:
Mild suggestion: could `SUBJECT == this` be used? Seeing as it's an enum?

Reply:
opportunity to merge. so this can be addressed in another branch.